### PR TITLE
{bio}[intel/2016b] miRDeep2 v2.0.0.8

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

depends on #4217, #4218, #4219, #4221

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2077.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fd621380ec60f5c087acff84b1014017 for a full test report.

File: easybuild/easyconfigs/m/miRDeep2/miRDeep2-2.0.0.8-intel-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'PackedBinary'
+
+name = 'miRDeep2'
+version = '2.0.0.8'
+altver = '_'.join(version.split('.'))
+
+homepage = 'https://www.mdc-berlin.de/8551903/en/'
+description = "miRDeep2 is a completely overhauled tool which discovers microRNA genes by analyzing sequenced RNAs"
+
+source_urls = ['https://www.mdc-berlin.de/45995549/en/research/research_teams/systems_biology_of_gene_regulatory_elements/projects/miRDeep/']
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/m/miRDeep2/miRDeep2-2.0.0.8-intel-2016b.eb]: line too long (141 > 120 characters) (but probably acceptable in this case...)

File: easybuild/easyconfigs/m/miRDeep2/miRDeep2-2.0.0.8-intel-2016b.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+sources = ['mirdeep%s.zip' % altver]
+
+toolchain = {'name': 'intel', 'version': '2016b'}
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/m/miRDeep2/miRDeep2-2.0.0.8-intel-2016b.eb]: toolchain is usually higher up (I guess the style guide will clarify this)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2077.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6e5945c2600a154d6016d7168d6389c2 for a full test report.

**Comment by boegel** [comment]
Thanks for the thorough review @migueldiascosta!

**Pull request merged**